requirements.in

```diff
@@ -1,11 +1,25 @@
 pandas
 pip-tools
 presto-python-client
-pyodbc
 pytest
 pyyaml
 requests
 seaborn
 sqlalchemy
 sqlparse
-tinynetrc
\ No newline at end of file
+tinynetrc
+
+# This is a pre-built wheel of cTDS, which supports accessing MSSQL databases using the TDS protocol.
+# We're using this because we had problems downloading large amounts of data over ODBC.
+# If you're on a platform other than Linux then you'll have to install cTDS yourself:
+# https://zillow.github.io/ctds/install.html
+# https://pypi.org/project/ctds/
+#
+https://github.com/opensafely/ctds-binary/raw/b76175dd04baace2327518a78536d35bac6c7e2b/dist/ctds-1.12.0-cp37-cp37m-manylinux2014_x86_64.whl ; sys_platform == "linux"
+
+# We still use pyODBC in our tests because we use SQLAlchemy to set up our test
+# data and there's currently no SQLAlchemy adapter for cTDS.  I managed to get
+# one mostly working, but it tripped up on the hyphen separated field names in
+# the EMIS backend. Once these have been removed it might be worth trying
```

Comment:
I've just merged #280, so the hyphens are gone.

Reply:
Thanks. I've stuck my WIP in a [branch](https://github.com/opensafely/cohort-extractor/tree/remove-pyodbc) so we can come back to it later.